src/app/cluster/details/cluster/edit-provider-settings/vmware-cloud-director-provider-settings/template.html

```diff
@@ -0,0 +1,84 @@
+<!--
+Copyright 2022 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<form [formGroup]="form"
+      fxLayout="column"
+      fxLayoutGap="8px">
+  <mat-form-field fxFlex>
+    <mat-label>Username</mat-label>
+    <input matInput
+           [formControlName]="Controls.Username"
+           [name]="Controls.Username"
+           type="text"
+           autocomplete="off"
+           required>
+    <mat-error *ngIf="form.get(Controls.Username).hasError('required')">
+      Username is <strong>required</strong>.
+    </mat-error>
+  </mat-form-field>
+
+  <mat-form-field fxFlex>
+    <mat-label>Password</mat-label>
+    <input matInput
+           [formControlName]="Controls.Password"
+           [name]="Controls.Password"
+           type="password"
+           autocomplete="off"
+           required>
+    <mat-error *ngIf="form.get(Controls.Password).hasError('required')">
+      Password is <strong>required</strong>.
+    </mat-error>
+  </mat-form-field>
+
+  <mat-form-field fxFlex>
+    <mat-label>Organization</mat-label>
+    <input matInput
+           [formControlName]="Controls.Organization"
+           [name]="Controls.Organization"
```

Comment:
We don't need `name` when using reactive forms

Reply:
agreed! both are different 1 is `HTML` element name and other Control binded with reactive forms